src/CoreFoundation/CFBundle.cs

```diff
@@ -581,21 +581,23 @@ public void UnloadExecutable ()
 		}
 
 #if MONOMAC
-#if !NET
+#if NET
+		[SupportedOSPlatform ("macos11.0")]
+		[UnsupportedOSPlatform ("maccatalyst")]
```

Comment:
Looks like the maccatalyst version numbers fell off here as well. Was that intended?

Reply:
I believe so, it is mirroring ` 		[NoMacCatalyst]`.